docs/api/migration-guides/_toc.json

```diff
@@ -27,6 +27,10 @@
         "title": "Migrate to the Qiskit Runtime V2 primitives",
         "url": "/api/migration-guides/v2-primitives"
       },
+      {
+        "title": "Migrate to local simulators",
```

Comment:
do you think we should make the title more specific? Like "Migrate from cloud simulators to local simulators"? I'm just wondering if people might miss the concept about the cloud simulators being removed